src/libs/actions/Policy/DistanceRate.ts

```diff
@@ -126,6 +126,48 @@ function enablePolicyDistanceRates(policyID: string, enabled: boolean) {
         ],
     };
 
+    if (!enabled) {
+        const policy = allPolicies?.[`${ONYXKEYS.COLLECTION.POLICY}${policyID}`];
+        const customUnitID = Object.keys(policy?.customUnits ?? {})[0];
+        const customUnit = customUnitID ? policy?.customUnits?.[customUnitID] : undefined;
+
+        const rateEntries = Object.entries(customUnit?.rates ?? {});
+        // find the rate to be enabled after disabling the distance rate feature
+        // first check the default rate
+        let rateEntryToBeEnabled = rateEntries.find((rate) => rate[1]?.name === 'Default Rate');
+        // if the default rate is not enabled/doesn't exist, we'll switch to the first enabled rate
```

Comment:
@dominictb noticed one case,

1. Create 3 distance rates
2. Disable the first one and leave the others enabled
3. Go offline
4. Disable and enable the feature
5. You'll notice the 2nd rate will remain enabled while the first one is disabled
6. Go online
7. The change will be reverted and only the first one will be re-enabled

I think we need to make sure that we re-enable the first distance rate, even if it was previously disabled.

https://github.com/Expensify/App/assets/75031127/e88365e3-8c9e-4e3d-9377-8acc9e886971